frontend/app_flowy/lib/workspace/presentation/home/home_layout.dart

```diff
@@ -37,6 +38,7 @@ class HomeLayout {
     }
 
     homePageLOffset = showMenu ? menuWidth : 0.0;
+    menuSpacing = showMenu ? 0 : 80.0;
```

Comment:
I suppose this space only shows on macOS. Maybe you need `Platform.isMacOS`.

Reply:
@vincentdchan done!